src/org/parosproxy/paros/view/View.java

```diff
@@ -1048,6 +1049,22 @@ public void displayMessage(Message message) {
         }
     }
 
+    public void addRequestPanelMessageChangedListener(MessageChangedListener messageChangedListener){

```

Comment:
Lets expose this through the `ExtensionHookView` instead, less changes for the add-ons that use it.